benchmarks/k8s/docker-images/Dockerfile.agent-2.7

```diff
@@ -0,0 +1,5 @@
+FROM python:2.7
```

Comment:
To be on the safe side / for consistency, we should probably also specify patch version, aka ``2.7.17`` or similar.

Reply:
Done.